diff --git a/src/main/java/mediathek/tool/GuiFunktionenProgramme.java b/src/main/java/mediathek/tool/GuiFunktionenProgramme.java
index 7d0283110..023a5f474 100644
--- a/src/main/java/mediathek/tool/GuiFunktionenProgramme.java
+++ b/src/main/java/mediathek/tool/GuiFunktionenProgramme.java
@@ -48,7 +48,7 @@ public class GuiFunktionenProgramme {

     private static final ArrayList<String> winPfade = new ArrayList<>();
     private static final Logger logger = LogManager.getLogger();
-    private static final String PFAD_LINUX_VLC = "/usr/bin/vlc";
+    private static final String PFAD_LINUX_VLC = "/app/bin/vlc";
     private static final String PFAD_MAC_VLC = "/Applications/VLC.app/Contents/MacOS/VLC";
     private static final String PFAD_WIN = "\\VideoLAN\\VLC\\vlc.exe";
     /**
